types.go

```diff
@@ -656,9 +684,9 @@ type PermissionLevelWeight struct {
 
 type Authority struct {
 	Threshold uint32                  `json:"threshold"`
-	Keys      []KeyWeight             `json:"keys,omitempty"`
-	Accounts  []PermissionLevelWeight `json:"accounts,omitempty"`
-	Waits     []WaitWeight            `json:"waits,omitempty"`
+	Keys      []KeyWeight             `json:"keys"`
+	Accounts  []PermissionLevelWeight `json:"accounts"`
+	Waits     []WaitWeight            `json:"waits"`
```

Comment:
Reasoning for removing the `omitempty`?

Reply:
Ok but we are still not aligned with the response since `null` is not `[]` (but to achieve it, we need to always instantiate slices to an empty slice, and the Go JSON serializer as no option to serialize `nil` slices to `[]`).

I'm kind of hesitant to change it although I'm pretty sure it has normally no impacts. 

Did the change help make tests pass?

Reply:
By https://github.com/eoscanada/eos-go/pull/191#discussion_r1016782712 , let me try to contain to assert as same between nil and empty array

Reply:
https://github.com/eoscanada/eos-go/pull/191/commits/ca5b8435b0ca0323c521abc3d07eb6985015a0f6